.github/workflows/nemo_tests.yml

```diff
@@ -122,6 +134,18 @@ jobs:
         tail -n 1 output.txt | grep -q "S_max:  0.407622"
         grep -A 1 "Elapsed Time" output.txt
         echo  $GITHUB_REF_NAME $GITHUB_SHA $(grep -A 1 "Elapsed Time" output.txt | head -n 2 | tail -n 1) >> ${NEMO_DIR}/performance_history_openacc_kernels_gpu
+        # Upload the results into a Github Gist
```

Comment:
Python script again?

Reply:
oh, good find, the github action looks much cleaner!